### PR TITLE
fix(DB/SmartScript): Fix Sand Sifter and Hai'shulud despawning after 1 minute

### PR DESCRIPTION
**Pull request by Samsequel**
## Changes Proposed:
-  checks ooc time instead

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/16458
- closes https://github.com/chromiecraft/chromiecraft/issues/5514

## Tests Performed:

- tested

## How to Test the Changes:

1. see linked issues
 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/fumper.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+--
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` IN (22039, 22482); 
+DELETE FROM `smart_scripts` WHERE (`entryorguid` IN (22038, 22482) AND (`source_type` = 0) AND (`id` IN (12));
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/fumper.sql:3]: extra parentheses
```suggestion
DELETE FROM `smart_scripts` WHERE `entryorguid` IN (22038, 22482) AND (`source_type` = 0) AND (`id` IN (12));
```

**Comment by Gultask** [comment]
Hmm these are spells they're summoned from.
w/ DurationIndex 4 which is 2mins

If you leave them be won't they despawn by themselves? Without any SAI rows, that is

**Comment by Samsequel** [comment]
> Hmm these are spells they're summoned from. w/ DurationIndex 4 which is 2mins
> 
> If you leave them be won't they despawn by themselves? Without any SAI rows, that is

you are right, they despawn after 2 minutes now

**Comment by Gultask** [comment]
In Combat, too?

**Comment by Samsequel** [comment]
> In Combat, too?

yep

**Comment by Gultask** [comment]
There isn't a way unless we hack it worse than it is now

**Comment by Samsequel** [comment]
> There isn't a way unless we hack it worse than it is now
> 
> Imo keep it 2mins

so remove the whole smartaction instead?

**Comment by Gultask** [comment]
ye

File: data/sql/updates/pending_db_world/fumper.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+--
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` IN (22039, 22482); 
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (22038, 22482) AND (`source_type` = 0) AND (`id` IN (12));
+
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/fumper.sql:4]:
```suggestion
--
DELETE FROM `smart_scripts` WHERE `entryorguid` IN (22038, 22482) AND (`source_type` = 0) AND (`id` IN (12));
```

**Review by Gultask — APPROVED** [review]

**Pull request merged**